src/react-components/settings-menu.js

```diff
@@ -430,14 +438,17 @@ export default class SettingsMenu extends Component {
     return (
       <div>
         {!this.props.showAsOverlay && (
-          <FontAwesomeIcon
-            icon={faBars}
+          <div
+            role="button"
+            aria-label="settings menu"
             onClick={() => this.setState({ expanded: !this.state.expanded })}
             className={classNames({
               [rootStyles.cornerButton]: true,
               [rootStyles.cornerButtonSelected]: this.state.expanded
             })}
-          />
+          >
+            <FontAwesomeIcon icon={faBars} />
```

Comment:
Does the change to nesting affect the styling at all? In general a best practice to nest the `FontAwesomeIcon`s instead of styling/making them clickable directly?

Reply:
I think the styling just happened to work out in this case, I do think it's bad practice to make the icon clickable, especially since these FontAwesomeIcons don't accept ARIA attributes, and they are directly rendered as `<svg>` tags.